cmd/immuclient/immuc/getcommands_test.go

```diff
@@ -172,6 +102,6 @@ func TestGetByRevision(t *testing.T) {
 	require.NoError(t, err)
 	require.Contains(t, msg, "value1")
 
-	msg, err = ic.Imc.Get([]string{"key@notarevision"})
+	_, err = ic.Imc.Get([]string{"key@notarevision"})
 	require.Error(t, err)
```

Comment:
You might want to `require.ErrorIs(t, err, ErrExpectedError)` or `require.EqualError(t, err, "some error")` or `require.ErrorContains(t, err, "must contain")` to make sure you get the expected error

Reply:
That will be done in another task